flow_action_components/GetPicklistValues/yarn.lock

```diff
@@ -0,0 +1,221 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Is this file needed? In the past, we've had maintenance headaches from lock files of this site when the versions change, security issues are found, etc...

Reply:
Thanks so much for adding these improvements, @renatoliveira!

Reply:
my preference would be to remove the prettier aspect. We've just had too many little problems with references to external libraries and lock files and I'd like to keep the repo internally consistent where possible.

Reply:
Removed the file. Maybe it is a good idea to add a "contributing" section where this is noted so other people don't try and commit it the same way I did initially.

Reply:
You're absolutely right. I wish I had a couple of volunteers to tune our content for developers interested in contributing...

I'll get this merged and packaged soon